core/src/main/java/org/apache/iceberg/rest/ResourcePaths.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+import java.util.Map;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.relocated.com.google.common.base.Joiner;
+
+public class ResourcePaths {
```

Comment:
Do we want to make this `V1ResourcePaths` or just deal with the change to `V2` if / when there is a V2?

Reply:
We're not sure how we're going to evolve them just yet, so I just hid version entirely in this class. We can extend that way later.

Reply:
Sounds good to me. When I was working on similar, having `V1` in the class name was rather ugly. So I actually prefer to avoid it if there’s a reasonable argument against it, which you’ve provided. Thanks.